### ISSUE TITLE
question about downloading crates is confusing

### ISSUE THREAD

**Issue by programmerjake** [rust-lang/surveys#314]
when i first read it, it took me a bit to figure out you specifically meant registries and not git dependencies, since i overlooked the "registry" word at first.
https://github.com/rust-lang/surveys/blob/3c169fb46bf6fccfdf6077394fc7a51d8353a1ea/surveys/2024-annual-survey/questions.md?plain=1#L487-L494

I think you may want to reword it to explicitly exclude git dependencies or other dependencies that don't use crates.io-like protocols otherwise people like me would have selected "I use a custom/local/company registry" since I use git dependencies and those dependencies' code doesn't come from crates.io.

**Comment by programmerjake** [comment]
or maybe add an option for git dependencies

**Comment by apiraino** [comment]
hey @programmerjake thanks for your comment.

Would this set of options be clearer?
```
- I use crates.io
- I use a custom/local/company crate registry
- I use a mirror of crates.io
- I download dependencies from git repositories (or other non crate-based registry)
- I don't know
```

(my wording is not perfect, feel free to improve 🙂 )

In any case I think we'll probably keep this suggestion in mind for next year since we have just closed edits for the survey 2024 and translations are finalized.

**Comment by programmerjake** [comment]
> hey [@programmerjake](https://github.com/programmerjake) thanks for your comment.
> 
> Would this set of options be clearer?

yes, it's much better!

> (my wording is not perfect, feel free to improve 🙂 )

yeah, since it's not going in this year maybe it would be best to leave a TODO there, so everyone can come up with better phrasing for next year.

> In any case I think we'll probably keep this suggestion in mind for next year since we have just closed edits for the survey 2024 and translations are finalized.

yeah, I figured I was too late so I didn't actually report it when i first found the issue, i reported it after being reminded again a few days later.
